bcos-sync/bcos-sync/BlockSync.cpp

```diff
@@ -42,6 +42,7 @@ BlockSync::BlockSync(BlockSyncConfig::Ptr _config, unsigned _idleWaitMs)
     m_downloadingTimer->registerTimeoutHandler(boost::bind(&BlockSync::onDownloadTimeout, this));
     m_downloadingQueue->registerNewBlockHandler(
         boost::bind(&BlockSync::onNewBlock, this, boost::placeholders::_1));
+    m_nodeTimeMaintenance = std::make_shared<NodeTimeMaintenance>();
```

Comment:
Pls  init `NodeTimeMaintenance` when init `BlockSync` and share this object with the sealer module.